scripts/brewkit/fix-machos.rb

```diff
@@ -41,7 +41,12 @@ def type
 
 class Fixer
   def initialize(file)
-    @file = MachO::MachOFile.new(file)
+    begin
+      @file = MachO::MachOFile.new(file)
+    rescue MachO::MachOError
+      # Fall back attempt, as some files are fat binaries
+      @file = MachO::FatFile.new(file)
```

Comment:
this will hide other errors. Instead determine if the file is a fat-file first.

Reply:
Well, der. [MachO::open()](https://github.com/Homebrew/ruby-macho/blob/3fd39143c2827b7dd6721a1f70e059e5dd0cf8ae/lib/macho.rb#L17-L43) does that by default.